lxd/network/driver_physical.go

```diff
@@ -29,26 +29,158 @@ func (n *physical) DBType() db.NetworkType {
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=mtu)
+		//
+		// ---
+		//  type: integer
+		//  shortdesc: MTU of the new interface
+		"mtu": validate.Optional(validate.IsNetworkMTU),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=vlan)
+		//
+		// ---
+		//  type: integer
+		//  shortdesc: VLAN ID to attach to
+		"vlan": validate.Optional(validate.IsNetworkVLAN),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=gvrp)
+		// This option specifies whether to register the VLAN using the GARP VLAN Registration Protocol.
+		// ---
+		//  type: bool
+		//  defaultdesc: `false`
+		//  shortdesc: Whether to use GARP VLAN Registration Protocol
+		"gvrp": validate.Optional(validate.IsBool),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=maas.subnet.ipv4)
+		//
+		// ---
+		//  type: string
+		//  condition: IPv4 address; using the `network` property on the NIC
+		//  shortdesc: MAAS IPv4 subnet to register instances in
+		"maas.subnet.ipv4": validate.IsAny,
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=maas.subnet.ipv6)
+		//
+		// ---
+		//  type: string
+		//  condition: IPv6 address; using the `network` property on the NIC
+		//  shortdesc: MAAS IPv6 subnet to register instances in
+		"maas.subnet.ipv6": validate.IsAny,
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=ipv4.gateway)
+		// Use CIDR notation.
+		// ---
+		//  type: string
+		//  condition: standard mode
+		//  shortdesc: IPv4 address for the gateway and network
+		"ipv4.gateway": validate.Optional(validate.IsNetworkAddressCIDRV4),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=ipv6.gateway)
+		// Use CIDR notation.
+		// ---
+		//  type: string
+		//  condition: standard mode
+		//  shortdesc: IPv6 address for the gateway and network
+		"ipv6.gateway": validate.Optional(validate.IsNetworkAddressCIDRV6),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=ipv4.ovn.ranges)
+		// Specify a comma-separated list of IPv4 ranges in FIRST-LAST format.
+		// ---
+		//  type: string
+		//  shortdesc: IPv4 ranges to use for child OVN network routers
+		"ipv4.ovn.ranges": validate.Optional(validate.IsListOf(validate.IsNetworkRangeV4)),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=ipv6.ovn.ranges)
+		// Specify a comma-separated list of IPv6 ranges in FIRST-LAST format.
+		// ---
+		//  type: string
+		//  shortdesc: IPv6 ranges to use for child OVN network routers
+		"ipv6.ovn.ranges": validate.Optional(validate.IsListOf(validate.IsNetworkRangeV6)),
+		// lxdmeta:generate(entities=network-physical; group=network-conf; key=ipv4.routes)
+		// Specify a comma-separated list of IPv4 CIDR subnets that can be used with the child OVN network's {config:option}`network-bridge-network-conf:ipv4.routes.external` setting.
```

Comment:
```suggestion
		// Specify a comma-separated list of IPv4 CIDR subnets that can be used with the child OVN network's {config:option}`device-nic-ovn-device-conf:ipv4.routes.external` setting.
```

No idea how I managed to copy&paste this link here ... I actually wrote that I'm linking to the device option. :smile: 
Let's hope this works now.

Reply:
fixed